CHANGELOG.md

```diff
@@ -6,6 +6,15 @@ and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.
 
 ## [Unreleased](https://github.com/pusher/chatkit-android/compare/1.0.2...HEAD)
 
+## [1.0.3](https://github.com/pusher/chatkit-android/compare/1.0.2...1.0.3)
+
+### Changed
+
+- Removed the `userId` constructor argument in favour of newly added `queryParams` argument
+  to keep consistent with other client libraries.
+- Removed the `authData` constructor argument to be in line with other client SDK's.
+- Both of the above are **breaking** changes.
```

Comment:
Then please put them under a `### Breaking` header, and bump the middle version number (or actually, just remove the release version header and leave it as "Unreleased", as this branch might wait for a better reason to break the interface.

Reply:
:+1: